integration/heapster_api_test.go

```diff
@@ -268,24 +268,28 @@ func runHeapsterMetricsTest(fm kubeFramework, svc *kube_api.Service) error {
 	for _, ts := range timeseries {
 		// Verify the relevant labels are present.
 		// All common labels must be present.
-		for _, label := range core.CommonLabels() {
-			if label == core.LabelContainerBaseImage && !isContainerBaseImageExpected(ts) {
-				continue
-			}
-			_, exists := ts.Labels[label.Key]
-			if !exists {
-				return fmt.Errorf("timeseries: %v does not contain common label: %v", ts, label)
-			}
-		}
 		podName, podMetric := ts.Labels[core.LabelPodName.Key]
-		if podMetric {
-			for _, label := range core.PodLabels() {
+
+		/*
+			TODO(mwielgus): Uncoment once all labels are populated from POD watch
```

Comment:
Without having made it through the actual code changes, commenting out an existing test scares me. Does this mean the new API output won't contain everything the old API output does?